app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
-  devise :database_authenticatable, :registerable, :confirmable, :recoverable, stretches: 12
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :confirmable, :trackable, :validatable
+  devise :database_authenticatable, :registerable, :recoverable, stretches: 12
```

Comment:
How about de-duplicating these two lines?  Ideally we'd have one line that starts with `devise`, followed by a unique list of all the modules we want.

Reply:
It oddly doesn't show on my end?